imports/plugins/core/accounts/server/no-meteor/schemas/account.graphql

```diff
@@ -331,6 +355,12 @@ extend type Mutation {
     input: SetAccountProfileCurrencyInput!
   ): SetAccountProfileCurrencyPayload
 
+  "Set the preferred language for an account"
+  setAccountProfileLanguage(
+    "Mutation input"
+    input: SetAccountProfileLanguageInput!
+  ): SetAccountProfileLanguagePayload
```

Comment:
Do `SetAccountProfileLanguagePayload!` so the client knows it will never be `null`.